src/components/Initiatives/InitiativeDetailsPage/InitiativeDetailsPage.tsx

```diff
@@ -91,13 +91,15 @@ export const InitiativeDetailsPage = () => {
     setIsFilterDialogOpen(false);
   };
 
+  const { singular: initiativeName } = useInitiativesCustomName();
+
   if (loading || loadingEntities) {
     return <Progress />;
   }
 
   if (error || initiative === undefined || filteredActionItems === undefined) {
     return (
-      <WarningPanel severity="error" title="Could not load Initiative.">
+      <WarningPanel severity="error" title={`Could not load ${capitalize(initiativeName)}.`}>
```

Comment:
Why capitalize here? We capitalize Initiative because it's IP but it's not safe to assume that customers want their term capitalized.
```suggestion
      <WarningPanel severity="error" title={`Could not load ${initiativeName}.`}>
```